core/field_textinput.js

```diff
@@ -234,16 +260,14 @@ Blockly.FieldTextInput.prototype.bindEvents_ = function(htmlInput) {
   htmlInput.onKeyDownWrapper_ =
       Blockly.bindEventWithChecks_(
           htmlInput, 'keydown', this, this.onHtmlInputKeyDown_);
-  // Bind to keyup -- trap Enter; resize after every keystroke.
+  // Bind to keyup -- resize after every keystroke.
   htmlInput.onKeyUpWrapper_ =
       Blockly.bindEventWithChecks_(
           htmlInput, 'keyup', this, this.onHtmlInputChange_);
   // Bind to keyPress -- repeatedly resize when holding down a key.
   htmlInput.onKeyPressWrapper_ =
       Blockly.bindEventWithChecks_(
           htmlInput, 'keypress', this, this.onHtmlInputChange_);
-  htmlInput.onWorkspaceChangeWrapper_ = this.resizeEditor_.bind(this);
-  this.workspace_.addChangeListener(htmlInput.onWorkspaceChangeWrapper_);
```

Comment:
Why did you remove this?  I think you need it to make sure that the inline editor floats to the right place if you resize the window or workspace while you're editing this text.

Reply:
If that's what it was supposed to do it doesn't do that, the workspace doesn't support resize/move/zoom events yet, plus window resize calls hideChaff. As far as I can tell it's just code accretion.

I'll revert it and create an issue to further investigate.